lib/Invocation.js

```diff
@@ -267,19 +267,23 @@ function prepareNextInvocation() {
       try {
         const result = job.invoke(cinv.fireDate instanceof CronDate ? cinv.fireDate.toDate() : cinv.fireDate);
         job.emit('run');
+        job.running += 1;
 
         if (result instanceof Promise) {
-          result.then(function(value) {
+          result.then(function () {
```

Comment:
value is lost here

Reply:
It was an old PR, so there was a little difference. just solved it